content/news/2015/02/2015-02-11-new-nih-clinical-trials-media-player-now-available.md

```diff
@@ -10,7 +10,6 @@ topics:
   - cdc
   - centers-for-disease-control-and-prevention
   - national-institutes-of-health
```

Comment:
```suggestion
  - emerging-tech
```

Reply:
"emerging tech" is stuff like AI, VR/AR, new technologies